framework/audio/engine/internal/audioengine.h

```diff
@@ -60,19 +59,17 @@ class AudioEngine : public IAudioEngine
     void setPan(handle h, float val) override;
     void setPlaySpeed(handle h, float speed) override;
 
+    async::Notification playCallbackCalled() const override;
+
+    // internal
+    void onPlayCallbackCalled();
```

Comment:
For what is this function needed?

Reply:
Maybe I didn't understand the question. You can examine the call stack and see what and how it is used. Above, I already wrote a comment about this functionality.